src/backend/joanie/tests/test_api_certificate.py

```diff
@@ -7,22 +7,22 @@
 
 from joanie.core.enums import PRODUCT_TYPE_CERTIFICATE
 from joanie.core.factories import (
-    CertificateDefinitionFactory,
     CertificateFactory,
+    IssuedCertificateFactory,
     OrderFactory,
     ProductFactory,
     UserFactory,
 )
 from joanie.tests.base import BaseAPITestCase
 
 
-class CertificateApiTest(BaseAPITestCase):
-    """Certificate API test case."""
+class IssuedCertificateApiTest(BaseAPITestCase):
+    """IssuedCertificate API test case."""
 
     def test_api_certificate_read_list_anonymous(self):
         """It should not be possible to retrieve the list of certificates for anonymous user"""
-        CertificateFactory.create_batch(2)
-        response = self.client.get("/api/certificates/")
+        IssuedCertificateFactory.create_batch(2)
+        response = self.client.get("/api/issued-certificates/")
```

Comment:
this is the point where you wonder if assertion wasn't a better name :fearful:

Reply:
And `CertificateClass` then?

Reply:
You'll have to move to `asserted-certificate` then :grimacing:

Reply:
Oh damn ! Good catch 🙏

Reply:
Yep, the endpoint is longer but finally this convention works fine and we will consistent accross our models and marion/howard models so let's go nope ?

Reply:
Honestly if we use `CertificateClass` instead of `Certificate`, I would prefer to use `CertificateDefinition` and `Certificate` 😅 

Yep, the endpoint is longer but finally this convention works fine and we will consistent across our models and marion/howard models so let's go nope ?

Reply:
I'm ok as long as this url is hidden but I feel it was clearer to call it a certificate and certificate definition (or certificate template?)

Reply:
@sampaccoud so can I resolve this conversation ? And if yes, could you review this PR again ?